rsa.go

```diff
@@ -278,8 +277,13 @@ func HashSignRSAPKCS1v15(priv *PrivateKeyRSA, h crypto.Hash, msg []byte) ([]byte
 }
 
 func VerifyRSAPKCS1v15(pub *PublicKeyRSA, h crypto.Hash, hashed, sig []byte) error {
-	if pub.withKey(func(pkey C.GO_EVP_PKEY_PTR) C.int {
-		size := C.go_openssl_EVP_PKEY_get_size(pkey)
+	if pub.withKey(func(pkey _EVP_PKEY_PTR) int32 {
+		var size int32
+		if vMajor == 1 {
+			size = go_openssl_EVP_PKEY_size(pkey)
+		} else {
+			size = go_openssl_EVP_PKEY_get_size(pkey)
+		}
```

Comment:
It seems like it would be less of a change if we added this mapping to the generated layer?

Also, this switch is done in more than one place: `evp.go` and `rsa.go`. That isn't *much*, but it's something.

Reply:
> It seems like it would be less of a change if we added this mapping to the generated layer?

Tricky, but done 😸.